public/javascript/pump/view.js

```diff
@@ -342,9 +342,21 @@
 
             view.$(".alert").alert();
         },
-        showError: function(msg) {
-            var view = this;
-            if (view.$(".alert").length > 0) {
+        showError: function(err) {
+            var view = this,
+                msg;
+
+            if (_.isString(err)) {
+                msg = err;
+            } else if (_.isObject(err)) {
```

Comment:
Shouldn't this be `_.ifError`?

Reply:
Not always the response is a `Error` instance, the ajax error response like 404, 0 (when server is down) the response is a `Object` instance